apps/bdd/package.json

```diff
@@ -29,7 +29,7 @@
     "@wdio/jasmine-framework": "^9.18.0",
     "@wdio/local-runner": "^9.18.4",
     "@wdio/spec-reporter": "^9.18.0",
-    "chromedriver": "^140.0.3",
+    "chromedriver": "^141.0.0",
```

Comment:
Just curious what’s happening you need to change it

Reply:
@dadiorchen I didn't physically change it. It might be because of yarn install or leftover old code? Either way both versions should work fine. I will change it to the old version.

Reply:
Don’t need, just ask, it is okay if everything is fine